dev/com.ibm.ws.webcontainer/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -170,4 +170,4 @@ throwExceptionWhenUnableToCompleteOrDispatch.name=Throw exception for not comple
 throwExceptionWhenUnableToCompleteOrDispatch.desc=Throw an illegal state exception when an asynchronous request cannot be completed or dispatched. The default is true. If the asynchronous request must complete or the dispatch method must return, even if the call does not succeed, set the property to false.
 
 getRealPathReturnsQualifiedPath.name=Always return a non-null path from getRealPath
-getRealPathReturnsQualifiedPath.desc=Return non-null paths from the ServletContext.getRealPath(String) method, even if no resource exists at the given path. The default is true. If applications expect the getRealPath method to return null when given a path for which no resource exists, set the property to false.
\ No newline at end of file
+getRealPathReturnsQualifiedPath.desc=Return non-null paths from the ServletContext.getRealPath(String) method, even if no resource exists at the given path. The default is true. If applications expect the getRealPath method to return null when given a path for which no resource exists, set the property to false.
```

Comment:
Change,
"Return non-null paths from the ServletContext.getRealPath(String) method, even if no resource exists at the given path. The default is true. If applications expect the getRealPath method to return null when given a path for which no resource exists, set the property to false."
to,
"Return non-null paths from the ServletContext.getRealPath(String) method, even if no resource exists at the path. The default is true. If applications expect the getRealPath method to return null when given a path for which no resource exists, set the property to false."